website/source/docs/auth/aws-ec2.html.md

```diff
@@ -1005,8 +1005,16 @@ in its identity document to match the one specified by this parameter.
 <dl class="api">
   <dt>Description</dt>
   <dd>
-    Creates a role tag on the role. Role tags provide an effective way to restrict the
-    capabilities that are set on the role.
+Creates a role tag on the role, which help in restricting the capabilities that
+are set on the role. Role tags are not tied to any specific ec2 instance unless
+specified explicitly using `instance_id` parameter. By default, role tags are
+designed to be used across all instances that satisfies the constraints on the
+role. Regardless of which instances have role tags on them, the restriction
+that the capabilities of role tags being a subset of capabilities on the role
+is strictly enforced. Note that, since adding and removing a tag is a widely
+distributed privilege, care needs to be taken to ensure that the instances are
+attached with correct tags to not let them gain more privileges than what were
+intended.
```

Comment:
I still think this is a bit wishy-washy. Mostly hard for people to understand how to use correctly. Maybe add one more sentence: "If a role tag is removed the capabilities inherited by the instance will be those defined on the role, so the role should never provide more capabilities than any given instance can be allowed to gain in a worst-case scenario."